state_mixin.py

```diff
@@ -14,6 +15,9 @@ class StateMixin(object):
     state_expr = ExpressionProperty(title='State Expression', default='{{$state}}')
     backup_interval = TimeDeltaProperty(title='Backup Interval',
                                         default={'seconds': 600})
+    load_from_persistence = BoolProperty(title="Load from persistence?", default=True,
```

Comment:
It probably doesn't need to. Are you interested in adding this feature now? I will put it in.